packages/host/app/components/matrix/payment-setup.gts

```diff
@@ -0,0 +1,259 @@
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+
+import InfoCircleIcon from '@cardstack/boxel-icons/info-circle';
```

Comment:
Why is this imported differently compared to other icons below? Ie. `import {... } from '@cardstack/boxel-ui/icons';`

Reply:
Actually this breaks my local build if I try to start the app. Says that module can not be found (I rebuild the addon and made a fresh install but still)

Reply:
@jurgenwerk  the icon is coming from boxel-icons ( Tabler icons ), you can try running [Quickstart Step 5](https://github.com/cardstack/boxel/blob/main/QUICKSTART.md) whether it's working again

```
cd ./packages/boxel-icons
pnpm build
```

Reply:
Ah thanks!